oneflow/core/functional/impl/math_functor.cpp

```diff
@@ -1170,7 +1170,17 @@ class SelectTopNFunctor {
   Maybe<TensorTuple> operator()(const TensorTuple& inputs, int32_t n) const {
     MutableAttrMap attr;
     JUST(attr.SetAttr<int32_t>("top_n", n));
+    std::vector<bool> require_grad;
+    std::vector<bool> is_leaf;
```

Comment:
在能确定长度的时候要提前声明长度，后面就不用 push_back，直接用 is_leaf.at(i) = xxx

```suggestion
    std::vector<bool> require_grad(n);
    std::vector<bool> is_leaf(n);
```